plugins/modules/site_workflow_manager.py

```diff
@@ -1088,15 +1179,35 @@ def update_floor(self, site_params):
             dict: The API response from the 'updates_a_floor' operation or None if an exception occurs.
         """
         response = None
-        units_of_measure = "feet"
+        units_of_measure = ["feet", "meters"]
+        rf_model = [
+            "Free Space",
+            "Outdoor Open Space",
+            "Cubes And Walled Offices",
+            "Indoor High Ceiling",
+            "Drywall Office Only"
+        ]
         try:
             self.log("Updating floor with parameters: {0}".format(site_params), "INFO")
             parent_name = site_params.get("site", {}).get("floor", {}).get("parentName")
             parent_id = self.get_parent_id(parent_name)
             site_params['site']['floor']['parentId'] = parent_id
```

Comment:
self.log("Retrieved parent ID: '{}' for parent name: '{}'".format(parent_id, parent_name), "DEBUG")

Reply:
Added